AltexSoft lab 2/scss/_media-query.scss

```diff
@@ -0,0 +1,442 @@
+@import "variables";
+@import "mixins";
+
+@media only screen and (min-width: $size-desktop-width) {
```

Comment:
You have only one media-query and 1440px - very strange size...

Almost all sections are 1200px width (1170px + 15px padding on the left and right). So definitely for desktop would be better to use `min-width: 1200px`.

But also you could add some more media-queries for smaller screens. Please open bootstrap (or any other popular css framework) and see what [breakpoints](https://getbootstrap.com/docs/4.1/layout/overview/) they use. Probably you won't need all of them in this project, but at least one more I recommend to add.